### PR TITLE
Added parent fields to child raml files

### PR DESCRIPTION
**Pull request by alex-ct**

### CODE REVIEW

**Comment by alex-ct** [comment]
Please note that the actual description will be added, but that is done with another script. This is done just as a first step to include parent fields into child RAML files.

File: types/cart-discount/CartDiscount.raml [hunk 1]
```diff
@@ -8,45 +8,76 @@ properties:
...
   createdAt:
     type: datetime
+    description: ''
   lastModifiedAt:
     type: datetime
+    description: ''
   lastModifiedBy?:
     type: LastModifiedBy
+    description: Present on resources updated after 1/02/2019 except for [events not
+      tracked](/http-api-client-logging#events-tracked).
```
[the remark this answers is not in the document]
Comment (jenschude) [reply types/cart-discount/CartDiscount.raml]: I would not invest time in the links atm as with a generated documentation the links would not be valid anymore

[the remark this answers is not in the document]
Comment (acbeni) [reply types/cart-discount/CartDiscount.raml]: Ah i see then i would suggest they are disabled.
meaning `[events not tracked]\(/http-api-client-logging#events-tracked)`
would be remplaces by `events not tracked`

File: types/cart/LineItem.raml [hunk 2]
```diff
@@ -6,38 +6,83 @@ type: object
...
     type: LineItemMode
+    description: ''
   custom?:
     type: CustomFields
+    description: ''
   shippingDetails?:
     type: ItemShippingDetails
+    description: |-
+      Container for line item specific address(es).
+      LineItem fields that can be used in query predicates: `productId`, `name`, `quantity`,
```
Comment (jenschude) [inline types/cart/LineItem.raml]: This is not belonging to the property

File: types/cart/LineItemDraft.raml [hunk 3]
```diff
@@ -22,17 +22,37 @@ properties:
...
+      `priceMode` to `ExternalPrice` LineItemPriceMode.
   externalTotalPrice?:
     type: ExternalLineItemTotalPrice
+    description: Sets the line item `price` and `totalPrice` to the given values and
+      sets the line item `priceMode` to `ExternalTotal` LineItemPriceMode.
   shippingDetails?:
     type: ItemShippingDetailsDraft
+    description: |-
+      Container for line item specific address(es).
+      The logic to select the price is described in LineItem Price selection.
```
Comment (jenschude) [inline types/cart/LineItemDraft.raml]: Again an leftover

File: types/common/CentPrecisionMoneyDraft.raml [hunk 4]
```diff
@@ -4,3 +4,10 @@
 displayName: CentPrecisionMoneyDraft
 type: TypedMoneyDraft
 discriminatorValue: centPrecision
+properties:
+  centAmount:
```
Comment (jenschude) [inline types/common/CentPrecisionMoneyDraft.raml]: These properties may not add informational value

**Comment by acbeni** [comment]
Looks very good to me 👍

**Pull request merged**